test/config/monitoring.yaml

```diff
@@ -43,7 +43,7 @@ spec:
     spec:
       containers:
       - name: zipkin
-        image: docker.io/openzipkin/zipkin:2.13.0
+        image: ghcr.io/openzipkin/zipkin:latest
```

Comment:
Maybe safer would be to use: `ghcr.io/openzipkin/zipkin:2` instead? If they release zipkin 3 that breaks something, this should save us from failure.

Reply:
ah, ok - that's good

will update